ResearchKit/Common/ORKOrderedTask.h

```diff
@@ -501,6 +501,37 @@ typedef NS_OPTIONS(NSUInteger, ORKPredefinedTaskOption) {
                               seriesLength:(NSInteger)seriesLength
                                    options:(ORKPredefinedTaskOption)options;
 
+
+typedef NS_ENUM(NSUInteger, PrebuiltSurveyType) {
+    SurveyTypeNottinghamEADL,
+    SurveyTypeStrokeImpactScale,
+    SurveyTypeEpworthSleepScale,
+    SurveyTypeIQCDE,
+    SurveyTypeRivermeadMobilityIndex
+};
+
```

Comment:
Refactor enum to include to ORK: `ORKPrebuiltSurveyType`. 
Also update all the enum options accordingly such as `ORKSurveyTypeNottinghamEADL`.